TIDALDL-PY/tidal_dl/download.py

```diff
@@ -54,11 +54,8 @@ def __getExtension__(url):
         return '.mp4'
     return '.m4a'
 
-def __getArtists__(array):
-    ret = []
-    for item in array:
-        ret.append(item.name)
-    return ret
+def __getArtists__(artists):
+    return ", ".join(map(lambda artist: artist.name, artists))

```

Comment:
this returns a `string` rather than `array` (**which only worked with `.m4a` and `.mp4`** because AIGPY tagHelper handles m4a/mp4 differently). I added a case for `.m4a` and `.mp4` below to use the old (array) method